assets/samples/edgeconnect/automation-workflows.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: dynatrace.com/v1alpha2
+kind: EdgeConnect
+metadata:
+  name: edge-connect-sample
+  namespace: dynatrace
+spec:
+  apiServer: "https://example.dev.apps.dynatracelabs.com"
+  replicas: 1
+
+  # The service account has to have the proper permissions that match the purpose of the
+  # workflow steps this EdgeConnect is used for.
+  serviceAccountName: edgeconnect-automation
+
+  # Enable K8S Automation for workflows
+  kubernetesAutomation:
+    enabled: true
+
+  oauth:
+    # OAuth client has to be created in Dynatrace account management
```

Comment:
can we link to docs for this?

Reply:
8bb546cca5e0c5f0c601d9eb380c4c95d6e0349e

Reply:
please remove the list of scopes from the sample. referencing them in the docs is enough

Reply:
we don't have the scopes as part of the dynakube api token, so we should not have them for edgeconnect either. the place to document how to create the token and what it should include is in the official documentation, not as part of the example yaml.

Reply:
For what it's worth, I find it more convenient from user PoV. But I've instead linked to the resp. docu page to make it very clear, which scopes are used.

Reply:
1978807a